docs/source/reference/optim/sgd.mdx

```diff
@@ -1,7 +1,20 @@
 # SGD
 
+Stochastic gradient descent (SGD) is a basic gradient descent optimizer to minimize loss given a set of model parameters and updates the parameters in the opposite direction of the gradient. The update is performed on a randomly sampled mini-batch of data from the dataset.
+
+bitsandbytes also supports momentum and Nesterov momentum to accelerate SGD by adding a weighted average of past gradients to the current gradient.
```

Comment:
This docs effort is so useful. It's these little details that makes the users more likely to find and use functionalities, really cool!